bot/exts/help_channels/_cog.py

```diff
@@ -580,7 +581,24 @@ async def notify_session_participants(self, message: discord.Message) -> None:
                 timestamp=message.created_at
             )
             embed.add_field(name="Conversation", value=f"[Jump to message]({message.jump_url})")
-            await message.author.send(embed=embed)
+
+            try:
+                await message.author.send(embed=embed)
+            except discord.errors.ForBidden:
+                log.trace(
+                    f"Failed to send helpdm message to {message.author.id}. DMs Closed/Blocked. "
+                    "Removing user from helpdm."
+                )
+                bot_commands_channel = self.bot.get_channel(Channels.bot_commands)
+                await _caches.help_dm.delete(message.author.id)
+                await bot_commands_channel.send(
+                    f"Hi, {message.author.mention} {constants.Emojis.cross_mark}. "
+                    f"Couldn't DM you helpdm message regarding {message.channel.mention} "
+                    "because your DMs are closed. helpdm has been automatically turned to off. "
+                    "to activate again type `!helpdm on`.",
+                    delete_after=10
```

Comment:
This integer should be replaced with `RedirectOutput.delete_delay` with `from bot.constants import RedirectOutput`

Reply:
Added it in the last commit https://github.com/python-discord/bot/pull/1642/commits/1b9500b5f7ea0c44ea5023bb264567b05bec1da6. Thanks!